### PR TITLE
Updated the link to meshery releases

### PR DESCRIPTION
**Pull request by sanjana-98**
Updated mesherys doc guide link from "https://github.com/layer5io/meshery/releases/tag/v0.3.9" to this:https://github.com/layer5io/meshery/releases/
 and also updated the command to "mesheryctl system start" from "./mesheryctl system start"

**Description**

This PR fixes #2867 

**Notes for Reviewers**

Signed-off-by: Sanjana<sanjana.balaji1@gmail.com
**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/guides/mesheryctl/working-with-mesheryctl.md [hunk 1]
```diff
@@ -160,11 +160,11 @@ Upgrade `mesheryctl` and run Meshery on Mac or Linux with this script:
 
 ### Installing the `mesheryctl` binary
 
-Download and unzip `mesheryctl` from the [Meshery releases](https://github.com/layer5io/meshery/releases/latest) page. Add `mesheryctl` to your PATH for ease of use. Then, execute:
+Download and unzip `mesheryctl` from the [Meshery releases](https://github.com/layer5io/meshery/releases/) page. Add `mesheryctl` to your PATH for ease of use. Then, execute:
 
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
- ./mesheryctl system start
+ mesheryctl system start
```
Comment (leecalcote) [inline docs/pages/guides/mesheryctl/working-with-mesheryctl.md:167]:
```suggestion
 ./mesheryctl system start
```

The `./` is intentional in the case of the Windows deployment. Unless the user has relocated the binary, the `./` is necessary.

**Review by leecalcote — APPROVED** [review]
Thanks, @sanjana-98. Looks good. 👍

**Comment by sanjana-98** [comment]
Yay! Thank you!

**Comment by leecalcote** [comment]
Whoo-hoo, @sanjana-98 🎈

**Pull request merged**